Application/Interfaces/ISimpleDbSet.cs

```diff
@@ -10,6 +10,18 @@ public interface ISimpleDbSet<TEntity> where TEntity : class
         CancellationToken CancellationToken { set; }
          
         Task<TEntity> FindAsync(object id);
+
+        async Task<TEntity?>? FindOrDefaultAsync(object id)
+        {
+            try
+            {
+                return await FindAsync(id);
+            }
+            catch (Exception e)
```

Comment:
Надо убрать ```e```, чтобы варнинг не выскакивал
```suggestion
            catch (Exception)
```

Reply:
зробив